integration-tests/common/common.go

```diff
@@ -51,7 +51,7 @@ type TestEnvDetails struct {
 type ChainDetails struct {
 	ChainName             string
 	ChainID               string
-	RPCUrl                string
+	RPCUrl                []string
```

Comment:
Should we make it plural?
```suggestion
	RPCUrls                []string
```